ui/src/app/pages/app/component.ts

```diff
@@ -169,9 +171,16 @@ export class AppComponent implements OnInit {
     });
   }
 
-  private loadConfig() {
+  private loadConfigAndErrorReporter() {
     this.serverConfigService.getConfig().subscribe((config) => {
       serverConfigStore.next(config);
+
+      const reporter = new StackdriverErrorReporter();
+      reporter.start({
+        key: config.publicApiKeyForErrorReports,
+        projectId: config.projectId,
+      });
+      stackdriverErrorReporterStore.set(reporter);
```

Comment:
can we go even simpler? what if we just call the old `setStackdriverErrorReporter` here?

Reply:
I could, but I'd rather have an `atom` than a regular global variable. What I could to do simplify more is have `reportError` grab the reporter from the reporter store rather than passing a `reporter` around.

Reply:
Oh cool. I didn't know we could do that from there. I think that would be a pretty good change if it means each logging component doesn't have to add a reporter store to their props.